modules/odc-invoke-serverless-function.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-getting-started.adoc
+
+:_content-type: PROCEDURE
+[id="odc-invoke-serverless-function_{context}"]
+= Testing a function in the web console
+
+You can test a deployed serverless function by invoking it in the *Developer* perspective of the {product-title} web console.
+
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Serving are installed on your {product-title} cluster.
+* You have logged in to the web console and are in the *Developer* perspective.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
```

Comment:
I think having created and deployed a function (below) implies this. Maybe it's unnecessary?